packages/web/src/components/ui/Card/Card.module.scss

```diff
@@ -0,0 +1,231 @@
+@import '~styles/_vars.scss';
+@import '~styles/responsive.scss';
+
+.card {
+  position: relative;
+  background: #ffffff;
+  border-radius: 24px;
+  padding: 24px;
+  box-shadow: $shadow;
+  width: 100%;
+  max-width: 600px;
+  min-height: 248px;
+  display: flex;
+  flex-direction: column;
+
+  @include lg {
+    padding: 20px;
+  }
+
+  @include mobile-and-tablet {
+    min-height: 204px;
+    box-shadow: $shadow-small;
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+}
+
+.icon {
+  flex-shrink: 0;
+  margin-right: 16px;
+  display: inline-flex;
+
+  img {
```

Comment:
I would like to avoid any tag name selectors. Let's use only class names and pseudo elements.